src/game/output.c

```diff
@@ -723,20 +723,13 @@ void Output_DrawScreenLine(
 }
 
 void Output_DrawScreenBox(
-    int32_t sx, int32_t sy, int32_t w, int32_t h, RGBA8888 col,
-    int32_t thickness)
-{
-    for (int i = 0; i < thickness; i++) {
-        // Top
-        Output_DrawScreenLine(sx - i, sy - i, w + 1 + (i * 2), 0, col);
-        // Right
-        Output_DrawScreenLine(w + sx + 1 + i, sy - i, 0, h + 1 + (i * 2), col);
-        // Left
-        Output_DrawScreenLine(sx - i, h + sy + 1 + i, 0, (-i * 2) - 1 - h, col);
-        // Bottom
-        Output_DrawScreenLine(
-            w + sx + 1 + i, h + sy + i, -w - 1 - (i * 2), 0, col);
-    }
+    int32_t sx, int32_t sy, int32_t w, int32_t h, RGBA8888 colDark,
+    RGBA8888 colLight, int32_t thickness)
```

Comment:
The argument names do not adhere to the coding convention
@walkawayy have you tested this?

Reply:
Yes I did some testing on this and it looks really good. I'll post some screenshots later. It works well even at low resolutions.